test/clt-tests/http-interface-tests/sql-mode-raw-endpoint-tests.rec

```diff
@@ -0,0 +1,506 @@
+}
+},
+{
+"Create Table": {
+"type": "string"
+}
+}
+],
+"data": [
+{
+"Table": "t",
+"Create Table": "CREATE TABLE t (\nid bigint,\nvalue text,\nvalue_attr string attribute\n) min_prefix_len='3' min_infix_len='3'"
+}
+],
+"total": 1,
+"error": "",
+"warning": ""
+}
+]
+––– input –––
+curl -s -X POST "http://localhost:9308/sql?mode=raw" -d "SELECT * FROM t" | jq .
+––– output –––
+[
+{
+"columns": [
+{
+"id": {
+"type": "long long"
+}
+},
+{
+"value": {
+"type": "string"
+}
+},
+{
+"value_attr": {
+"type": "string"
+}
+}
+],
+"data": [
+{
+"id": 1,
+"value": "example",
+"value_attr": "example"
+},
+{
+"id": 2,
+"value": "test",
+"value_attr": "test"
+}
+],
+"total": 2,
+"error": "",
+"warning": ""
+}
+]
+––– input –––
+curl -s -X POST "http://localhost:9308/sql?mode=raw" -d "SHOW TABLES" | jq .
+––– output –––
+[
+{
+"columns": [
+{
+"Table": {
+"type": "string"
+}
+},
+{
+"Type": {
+"type": "string"
+}
+}
+],
+"data": [
+{
+"Table": "t",
+"Type": "rt"
+}
+],
+"total": 1,
+"error": "",
+"warning": ""
+}
+]
+––– input –––
+curl -s -X POST "http://localhost:9308/sql?mode=raw" -d "DESCRIBE t" | jq .
+––– output –––
+[
+{
+"columns": [
+{
+"Field": {
+"type": "string"
+}
+},
+{
+"Type": {
+"type": "string"
+}
+},
+{
+"Properties": {
+"type": "string"
+}
+}
+],
+"data": [
+{
+"Field": "id",
+"Type": "bigint",
+"Properties": ""
+},
+{
+"Field": "value",
+"Type": "text",
+"Properties": "indexed stored"
+},
+{
+"Field": "value_attr",
+"Type": "string",
+"Properties": ""
+}
+],
+"total": 3,
+"error": "",
+"warning": ""
+}
+]
+––– input –––
+curl -s -X POST "http://localhost:9308/sql?mode=raw" -d "SELECT * FROM t WHERE MATCH('example')" | jq .
+––– output –––
+[
+{
+"columns": [
+{
+"id": {
+"type": "long long"
+}
+},
+{
+"value": {
+"type": "string"
+}
+},
+{
+"value_attr": {
+"type": "string"
+}
+}
+],
+"data": [
+{
+"id": 1,
+"value": "example",
+"value_attr": "example"
+}
+],
+"total": 1,
+"error": "",
+"warning": ""
+}
+]
+––– input –––
+curl -s -X POST "http://localhost:9308/sql?mode=raw" -d "SHOW META" | jq .
+––– output –––
+[
+{
+"columns": [
+{
+"Variable_name": {
+"type": "string"
+}
+},
+{
+"Value": {
+"type": "string"
+}
+}
+],
+"data": [
+{
+"Variable_name": "total",
+"Value": "0"
+},
+{
+"Variable_name": "total_found",
+"Value": "0"
+},
+{
+"Variable_name": "total_relation",
+"Value": "eq"
+},
+{
+"Variable_name": "time",
+"Value": "0.000"
+}
+],
+"total": 4,
+"error": "",
+"warning": ""
+}
+]
+––– input –––
+curl -s -X POST "http://localhost:9308/sql?mode=raw" -d "SELECT * FROM t WHERE MATCH('example') OPTION ranker=bm25" | jq .
+––– output –––
+[
+{
+"columns": [
+{
+"id": {
+"type": "long long"
+}
+},
+{
+"value": {
+"type": "string"
+}
+},
+{
+"value_attr": {
+"type": "string"
+}
+}
+],
+"data": [
+{
+"id": 1,
+"value": "example",
+"value_attr": "example"
+}
+],
+"total": 1,
+"error": "",
+"warning": ""
+}
+]
+––– input –––
+curl -s -X POST "http://localhost:9308/sql?mode=raw" -d "SELECT * FROM t WHERE MATCH('exa*')" | jq .
```

Comment:
Let's think about adding buddy queries to each test we do, because currently we validate only plain requests for daemon, but most interest is about Buddy.